src/com/tigersale/userInterface/inventoryManager/RegisterInventoryManagerView.java

```diff
@@ -0,0 +1,102 @@
+package com.tigersale.userInterface.inventoryManager;
+
+import com.sun.corba.se.impl.io.TypeMismatchException;
+import com.tigersale.database.InventoryManagerTable;
+import com.tigersale.userInterface.AbstractView;
+
+import java.sql.Date;
+import java.util.Scanner;
+
+/**
+ * @author James Haller
+ */
+public class RegisterInventoryManagerView extends AbstractView {
+    public RegisterInventoryManagerView(Scanner scanner) {
+        super(scanner);
+    }
+
+    public void runView() {
+        String username;
+        String password;
+        int salary;
+        Date dateOfBirth;
+        String firstName;
+        String lastName;
+        String middleInitial;
+
+        // Get the username
+        System.out.println("Please type in your username.");
```

Comment:
This would be "their" username.  This is sort of like a manager hiring and registering a new person to the company.